datahub-web-react/src/app/settings/AccessTokens.tsx

```diff
@@ -174,12 +174,13 @@ export const AccessTokens = () => {
     const renderSearchResult = (entity: any) => {
         const {
             editableProperties,
-            info: { displayName },
+            info,
         } = entity;
+        const displayNameSearchResult = info?.displayName || editableProperties?.displayName || entity.username;
```

Comment:
cool I know this is an improvement for sure. can we just call `entityRegistry.getDisplayName(entity.type, entity)` here though?

Reply:
Sure, we can simplify it by using entityRegistry.getDisplayName(entity.type, entity). Updated as per your suggestion.